src/util/ViewAnimation.js

```diff
@@ -256,10 +256,10 @@ const FlyAnimation = {
    * Zoom to fit the given extent by using a "fly" animation.
    * @param {ol.View} view The `ol.View` of the map.
    * @param {ol.Extent} extent The destination extent to zoom to.
-   * @param {function(complete)} completionCallback An optional notification that the animation has completed.
+   * @param {function(complete)} [completionCallback] An optional notification that the animation has completed.
    * @param {Object} options Configuration object for the animation, supported attributes are:
-   * * {Number} duration An optional animation duration.
-   * * {Number} maxZoom An optional maximal zoom level.
+   * * {Number} [duration] An optional animation duration.
+   * * {Number} [maxZoom] An optional maximal zoom level.
```

Comment:
```suggestion
   * @param {Number} [options.duration] An optional animation duration.
   * @param {Number} [options.maxZoom] An optional maximal zoom level.
```

To conform to official [JSDoc Parameters with properties](https://jsdoc.app/tags-param)